isb_web/manage.py

```diff
@@ -204,7 +204,11 @@ async def login(request: starlette.requests.Request):
     Initiate OAuth2 login with ORCID
     """
     redirect_uri = request.url_for("auth")
-    return await oauth.orcid.authorize_redirect(request, redirect_uri)
+    # check if login is for annotation purpose, if so add query param 
+    if "annotation" in request.query_params and request.query_params["annotation"]=="true" :
+        redirect_uri += "?annotation=true"
```

Comment:
If we are accessing the `login` endpoint from throughput, we pass the `annotation=true` query param to the `auth` endpoint as well.